Makefile

```diff
@@ -56,37 +71,20 @@ ifneq ($(DEPLOY_REGION),)
 	$(eval LIMIT=$(LIMIT):&region_$(DEPLOY_REGION))
 endif
 
-	cd ansible && $(ENV) ansible-playbook \
-		--limit="$(LIMIT)" \
-		-e ansible_python_interpreter=/usr/bin/python3 \
-		-e package=$(PACKAGE) \
-		-e env=$(DEPLOY_ENV) \
-		-e "@$(CONFIG_OUTPUT_DIR)/$(DEPLOY_ENV).yml" \
-		-e downtime=$(DEPLOY_DOWNTIME) \
-		-e db_version=$(DEPLOY_DB_VERSION) \
-		-e rolling_update="${ROLLING_UPDATE}" \
-		deploy.yml
+	ANSIBLE_EXTRA_VARS=$(DEPLOY_EXTRA_VARS)
+
+deploy: ansible/deploy.yml
 
 manage-node: check-deploy-env secrets vault-config-$(DEPLOY_ENV)
 ifndef CMD
 	$(error CMD is undefined, supported commands: start, stop, restart, ping)
 endif
-	cd ansible && $(ENV) ansible-playbook \
-		--limit="tag_env_$(DEPLOY_ENV):&tag_role_aenode" \
-		-e ansible_python_interpreter=/usr/bin/python3 \
-		-e env=$(DEPLOY_ENV) \
-		-e "@$(CONFIG_OUTPUT_DIR)/$(DEPLOY_ENV).yml" \
-		-e db_version=0 \
-		-e cmd=$(CMD) \
-		manage-node.yml
 
-reset-net: check-deploy-env secrets vault-config-$(DEPLOY_ENV)
-	cd ansible && $(ENV) ansible-playbook \
-		--limit="tag_env_$(DEPLOY_ENV):&tag_role_aenode" \
-		-e ansible_python_interpreter=/usr/bin/python3 \
-		-e env=$(DEPLOY_ENV) \
-		-e "@$(CONFIG_OUTPUT_DIR)/$(DEPLOY_ENV).yml" \
-		reset-net.yml
+	ANSIBLE_EXTRA_VARS=$(MANAGE_NODE_EXTRA_VARS)
+
+manage-node: ansible/manage-node.yml
+
+reset-net: check-deploy-env secrets vault-config-$(DEPLOY_ENV) ansible/reset-net.yml
```

Comment:
`check-deploy-env secrets vault-config-$(DEPLOY_ENV)` not needed

Reply:
ffcbaee